circle.yml

```diff
@@ -137,6 +138,35 @@ test:
       do
         convert $dir/$x/*.png -append $dir/$x.png
       done
+      # Remove all directories in $CIRCLE_ARTIFACTS/img
+      # Currently the pieces aren't kept around. If it's
+      # desirable to keep them around, we should use
+      #    cp -r $dir $CIRCLE_ARTIFACTS/img_pieces
+      # The reason the image pieces aren't currently kept
+      # around is that it was leading to a problem and didn't
+      # want to sink too much time into the problem at the moment.
```

Comment:
haha, maybe be slightly more specific? IIRC it was that the number of images in our artifacts directory was increasing quadratically, wanna say that?